specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-03-01-preview/connectedClusters.json

```diff
@@ -0,0 +1,802 @@
+            "$ref": "./examples/DeleteClusterExample.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNamePathParam"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Resource deletion is in progress"
+          },
+          "204": {
+            "description": "The resource was not found but the request is well formed"
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Kubernetes/connectedClusters/{clusterName}/listClusterUserCredentials": {
+      "post": {
+        "summary": "Gets cluster user credentials of a connected cluster",
+        "operationId": "ConnectedCluster_ListClusterUserCredentials",
+        "description": "Gets cluster user credentials of the connected cluster with a specified resource group and name.",
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "ListClusterUserCredentialsExample": {
+            "$ref": "./examples/ConnectedClustersListClusterCredentialsResult.json"
+          },
+          "ListClusterUserCredentialsNonAadExample": {
+            "$ref": "./examples/ConnectedClustersListClusterCredentialsResultNonAad.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNamePathParam"
+          },
+          {
+            "$ref": "#/parameters/ClientProxy"
+          },
+          {
+            "$ref": "#/parameters/TokenAuth"
```

Comment:
This should be in the POST body, not the query params.  See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#resource-action-requests

Reply:
Hi, just to clarify, it is not actually the TokenAuth...it is actually a boolean query parameter which indicates if the authentication is gona be AAD auth or token auth. Hence, it is a query param..Please let me know if I still have to change it

Reply:
See "All parameters for the action request should be contained in the request body – and not in the URL. This matches the OData and Azure REST guidelines and will avoid impacting the authZ checks." in the RPC doc at the link above.  Please put it in the body

Reply:
okay thanks! 
I have two other important questions :
1.	We had AAD profile as a required property in our existing preview swagger but we have removed it from our new swagger version- Is this breaking change acceptable to you ?
2.	We would like to check in a merged swagger (merging our preview and GA api version) which we will be using to generate SDK, is it ok to check that in in the stable folder ?

Reply:
Removal of something from preview as you move to GA is fine.  Any swagger specs going into the stable folder need to be a non-preview API version.  Any changes you are ready to expose in a non-preview API version are OK to put in that folder under the appropriate new stable api version

Reply:
I see, thanks!

Reply:
Hi, need guidance on this- 
1. We have our ListclusterUsercredentials api which is present only in preview and not in GA, but, it will be present in our merged swagger(which will be used for generating sdk ), so is it ok to have this merged swagger in preview folder ? (or any other place?)
2. Should we remove both the query params "clientProxy" and "TokenAuth" and add to body ?

Reply:
Hi, I have made the changes accordingly..Please review and merge it

Reply:
hi @pilor , please mark the "requested changes" as resolved..It blocking the merging